src/v2/guide/reactivity.md

```diff
@@ -110,9 +110,9 @@ Vue.component('example', {
   methods: {
     updateMessage: function () {
       this.message = 'updated'
-      console.log(this.$el.textContent) // => 'not updated'
+      console.log(this.$el.textContent) // => 'không được cập nhật'
```

Comment:
Vì `this.message = 'updated'` nên `console.log` phải in ra `// 'not updated'` chứ sao in ra tiếng Việt được?